source/index.html.md

```diff
@@ -21,10 +23,19 @@ search: true
 
 # Introduction
 
-This API provides information about calls to action.  A call to action is
-represented by the event resource.  An event resource has two related
-resources: a contact and (optionally) a location.
+This API provides information about calls to action (CTAs).  It uses the data modelling
+outlined in the [Open Source Data Interface](https://opensupporter.github.io/osdi-docs/).
+All resources in the API are OSDI compliant.
 
-The API follows the [JSON API spec](http://jsonapi.org/).  Feature like
-pagination, filtering, and CRUDing resrouces are all implemented following the
+The CTA Aggregator does deviate from OSDI when it comes to data presentation. Whereas the
+OSDI standard leverages HAL+JSON](https://tools.ietf.org/html/draft-kelly-json-hal-05) to
+repsesent resources, the CTAAggrategor uses the [JSON API spec](http://jsonapi.org/).  
+Feature like pagination, filtering, and CRUDing resrouces are all implemented following the 
 conventions established in this spec.
+
+The CTA Aggregator has to top-level resources: Events and Advocacy Campaigns.
+An event is a call to action that asks people to show up a particular location.
+And advocacy campaign is a call to action that requests that people send an email or make a
+phone call.
+
+You can interact with the API directly or use the Ruby gem, located here: [https://github.com/Ragtagteam/cta-aggregator-client-ruby](https://github.com/Ragtagteam/cta-aggregator-client-ruby/).
```

Comment:
This feels a little bit unclear. Might be clearer to add a sentence clause that describes why one might want to use the Ruby gem instead of interacting with it directly?